mlrun/api/db/sqldb/models/models_mysql.py

```diff
@@ -532,7 +532,16 @@ class DatastoreProfile(Base, mlrun.utils.db.BaseModel):
         name = Column(String(255, collation=SQLCollationUtil.collation()))
         project = Column(String(255, collation=SQLCollationUtil.collation()))
         type = Column(String(255, collation=SQLCollationUtil.collation()))
-        body = Column(String(1024, collation=SQLCollationUtil.collation()))
+        object = Column("object", JSON)
```

Comment:
```suggestion
        _object = Column("object", JSON)
```
We want to treat this as protected member - clients should use the `full_object` accessor/setter properties you added.